### PR TITLE
add predictive policing section

### PR DESCRIPTION
**Pull request by ymaltsman**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +42,7 @@
         <li><a href="#stingray">Cell Site Simulators</a></li>
         <li><a href="#shotspotter">Shotspotter</a></li>
         <li><a href="#socmint">Social Media</a></li>
+		<li><a href="#cvrm">Predictive Policing</a></li>
```
Comment (b-meson) [inline index.html:45]: I think this indent shouldn't be here?

File: index.html [hunk 2]
```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
  </div>
 </section>
 
+<section>  
+  <div class="content" id="cvrm">
+    <h1>Predictive Policing</h1>
+    <p>Since 2012, the CPD has used a statistical model to predict which individuals are most likely to be involved in shooting incidents. The model, developed in collaboration with the Illinois Institute of Technology, was initially called the Strategic Subject List (SSL) and was later revised and renamed the Crime and Victimization Risk Model (CVRM). This model was quietly <a id="cvrmlink" href="https://igchicago.org/2020/01/23/advisory-concerning-the-chicago-police-departments-predictive-risk-models/">decommissioned</a> by the CPD in 2019. </p>
```
Comment (b-meson) [inline index.html:394]: s/shooting/violent crime.

File: index.html [hunk 3]
```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
...
+        <div class="col-md-6"><img src="img/CVRMimage.png" alt="Socmint" class="img-rounded"></div>
+    </div>
+ </div>
+</section>
+
+<section class="nonheading">  
+  <div class="content">
+  	<div class="row">
+    <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
+    <p>The CVRM was meant to play a key role in CPD’s new <a href="https://www.rand.org/pubs/research_reports/RR3242.html">Strategic Decision and Support Centers (SDSCs)</a>, real time crime centers which are now located in all but two of Chicago’s 22 districts. Officers could also view at risk individuals identified by the CVRM on a phone app.</p>
```
Comment (b-meson) [inline index.html:406]: I would say "real time crime centers"

File: index.html [hunk 4]
```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
...
+    <p>CPD received $3.8 million in federal grants to develop the SSL and CVRM.</p>
+    </div>
+	<div class="row">
+		<div class="col-md-6"></div>
+		<div class="col-md-6"><img src="img/SSL-Dash.jpg" style="width:100%"><h5>A screenshot of an individual's risk classification produced by the SSL. Obtained via FOIA request by <a href="https://southsideweekly.com/predictive-policing-long-road-transparency/">South Side Weekly</a>.</div>
+	</div>
+  	<div class="row">
+    <h3> <span class="glyphicon glyphicon-eye-open"></span> Capabilities </h3>
+    <p> The CVRM outputs a risk classification for anyone arrested in Chicago over the past four years (over 300,000 people). The model’s output is based on risk factors such as a subject’s age and involvement in various types of crimes, with more recent incidents being given a higher weight than older ones. Individuals’ scores are also influenced by the scores of co-arestees. IIT provided a <a href="https://home.chicagopolice.org/wp-content/uploads/2019/01/FACT-SHEET-Crime-and-Victimization-Risk-Model-1.pdf">fact sheet</a> which describes the model in greater detail.</p>
```
Comment (b-meson) [inline index.html:417]: This is not technically true. "The CVRM outputs a risk classification for anyone arrested in Chicago over the past four years (over 300,000 people)." The part about arrests is true but that's not the only people placed on the list. If you were the _victim_ of a violent crime, you were also put on the SSL. Also please include this statistic "56 percent of black men in the city ages 20 to 29 have an SSL score " 

```
The newest data release about the SSL showed that nearly 400,000 people were assigned scores on the list, using primarily eight variables: age, the number of times an individual has been shot, the number of times they were the victim of assault or battery, gang affiliation, arrests for violent offenses, drugs, or weapons, and whether someone is getting arrested more frequently over time.
```
https://www.chicagomag.com/city-life/August-2017/Chicago-Police-Strategic-Subject-List/

Comment (b-meson) [reply index.html:417]: I would also strongly caution about using IIT's "risk model 1" the SSL evolved over time and there are a number of risk models. It was supposed to be targeted to the "most likely to be violent" and then gradually expanded to included 400,000 names. The RAND corp study you mention later only studied v1 of the algorithm, which CPD disputes because they changed the model over time. (I think by the time it was decommissioned it was on version 7)

File: index.html [hunk 5]
```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
...
+    <p>CVRM contributed to the <a href="https://home.chicagopolice.org/information/violence-reduction-strategy-vrs/">Custom Notification Program</a>, whereby at risk individuals and families are contacted and visited, reminded of the risk of being involved in violent crime, and connected with social services. </p>
```
Comment (b-meson) [inline index.html:418]: CPD and the States attorney would explicitly give people _higher_ sentences if they attempted to do a notification (source http://directives.chicagopolice.org/directives/data/a7a57bf0-1456faf9-bfa14-570a-a2deebf33c56ae59.pdf?hl=true) 

Another concern is that an attempted notifications doesn't mean a person was reached (its been told to me that sometimes they would give notification to friends, neighbors or family). This is mentioned on page 4 of the OIG report.

File: index.html (hunk 4 above)
Comment (b-meson) [reply index.html:417]: CPD gave everyone with an arrest an SSL score but never purged it if charges where dropped.

**Comment by b-meson** [comment]
Numerous comments to review. @ymaltsman please review and let me know how you want to proceed. Feel free to add an invitation to chat on my calendar.

File: index.html (hunk 1 above)
Comment (ymaltsman) [reply index.html:45]: That's odd. I don't see an indent on my local version.

File: index.html (hunk 3 above)
Comment (ymaltsman) [reply index.html:406]: I said that the SDSCs are real time crime centers. Is it too wordy?

File: index.html (hunk 4 above)
Comment (ymaltsman) [reply index.html:417]: I actually think we're talking about two different RAND studies: (1) http://link.springer.com/article/10.1007/s11292-016-9272-0 from 2016, about v1, and (2) https://www.rand.org/pubs/research_reports/RR3242.html from 2019, about SDSCs in general, which talks about the CVRM so at least v5 or v6

**Comment by ymaltsman** [comment]
@b-meson updated the PR. Check it out when you get a chance

File: index.html [hunk 6]
```diff
@@ -387,6 +388,48 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
...
+	</div>
+  	<div class="row">
+    <h3> <span class="glyphicon glyphicon-eye-open"></span> Capabilities </h3>
+    <p> CPD’s PTV model outputs a risk classification for anyone arrested in Chicago over the past four years (over 300,000 people). The model’s output is based on risk factors such as a subject’s age and involvement in various types of crimes, with more recent incidents being given a higher weight than older ones. Individuals’ scores are also influenced by the scores of co-arestees. IIT provided a <a href="https://home.chicagopolice.org/wp-content/uploads/2019/01/FACT-SHEET-Crime-and-Victimization-Risk-Model-1.pdf">fact sheet</a> which describes the model in greater detail. Note, however, that the model and its inputs and outputs have changed over time. Previously, the model took into account gang affiliation based on the now defunct (although soon to be revamped) <a href="https://chicago.suntimes.com/2020/2/26/21155215/cpd-revamp-controversial-gang-database">gang database</a>.</p>
```
Comment (b-meson) [inline index.html:417]: nit but commas should go inside of the closing parenthesis... `.)`

File: index.html (hunk 5 above)
Comment (b-meson) [reply index.html:418]: Hi @ymaltsman I think we didn't address this point, that the SSL will be used for targeted enhancements.

**Comment by b-meson** [comment]
I think its just missing this quote, which I think is really important "56 percent of black men in the city ages 20 to 29 have an SSL score." then we can merge.

File: index.html [hunk 7]
```diff
@@ -387,6 +388,48 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
...
+	<h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privacy Concerns</h3>
+    <p> Although the PTV model’s developers claim that it does not take race and other social factors into account, <a href="https://www.chicagomag.com/city-life/August-2017/Chicago-Police-Strategic-Subject-List/">56%</a> of all Black males between 20 and 29 had an SSL score. The model is trained on historical arrest data which is skewed due to biased policing practices. An investigative report by <a href="https://www.propublica.org/article/machine-bias-risk-assessments-in-criminal-sentencing">ProPublica</a> from 2016 revealed how predictive policing algorithms tend to disproportionately target people of color.</p>
```
Comment (ymaltsman) [inline index.html:426]: @b-meson that quote actually is there on 426

**Comment by b-meson** [comment]
Merging via a squash.
